src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/RespondentTellSomethingElseHelper.java

```diff
@@ -64,7 +64,8 @@ public static String getDocumentRequest(CaseData caseData, String accessKey)
     }
 
     private static GenericTseApplicationTypeItem getCurrentGenericTseApplicationTypeItem(CaseData caseData) {
-        if (caseData.getGenericTseApplicationCollection() == null) {
+        if (caseData.getGenericTseApplicationCollection() == null 
```

Comment:
CollectionUtils.isEmpty() does the same but can refactor later

Reply:
sorted.